ghost/content.md

```diff
@@ -1,33 +1,58 @@
 # Ghost
 
-Ghost is a free and open source blogging platform written in JavaScript and distributed under the MIT License, designed to simplify the process of online publishing for individual bloggers as well as online publications.
+Ghost is a free and open source blogging platform written in JavaScript and distributed under the MIT Licence, designed to simplify the process of online publishing for individual bloggers as well as online publications.
 
 > [wikipedia.org/wiki/Ghost_(blogging_platform)](http://en.wikipedia.org/wiki/Ghost_%28blogging_platform%29)
 
 %%LOGO%%
 
 # How to use this image
 
+This will start a Ghost instance listening on the default Ghost port of 2368.
+
 ```console
-$ docker run --name some-ghost -d ghost
+$ docker run —d --name ghostdemo ghost
 ```
 
-This will start a Ghost instance listening on the default Ghost port of 2368.
+### Custom port
 
 If you'd like to be able to access the instance from the host without the container's IP, standard port mappings can be used:
 
 ```console
-$ docker run --name some-ghost -p 8080:2368 -d ghost
+$ docker run —d --name ghostdemo -p 3001:2368 ghost
 ```
 
-Then, access it via `http://localhost:8080` or `http://host-ip:8080` in a browser.
+Then, access it via `http://localhost:3001` or `http://host-ip:3001` in a browser.
+
+# Stateful
 
-You can also point the image to your existing content on your host:
+Mount your existing content. In this example we also use the Alpine base image.
+
+### Ghost 1.x.x
 
 ```console
-$ docker run --name some-ghost -v /path/to/ghost/blog:/var/lib/ghost ghost
+$ docker run —d --name ghostdemo -p 3001:2368 -v /path/to/ghost/blog:/var/lib/ghost/content/ ghost:1-alpine
```

Comment:
Please drop the trailing slash on `content/` and stick to the `some-ghost` naming (we use that convention fairly consistently across the official images descriptions).